ledger/src/blockstore_meta.rs

```diff
@@ -327,6 +330,34 @@ pub struct ProgramCost {
     pub cost: u64,
 }
 
+#[derive(Clone, Debug, Default, Deserialize, Serialize, PartialEq)]
+pub struct OptimisticSlotMetaV0 {
```

Comment:
In case we want to handle schema change in the future, we might need to also include the version information when we persist it in the database.

Reply:
It should already be handled. The DB stores the versioned struct:

```
impl TypedColumn for columns::OptimisticSlots {
    type Type = blockstore_meta::OptimisticSlotMetaVersioned;
}
```

The enum should be serialized and deserialized. Adding a new variant should just require adding a new version of the struct and wrapping it in a new enum version.